aten/src/ATen/functorch/BatchRulesLinearAlgebra.cpp

```diff
@@ -280,6 +280,62 @@ static void expect_at_least_rank(
       rank, " dimensions instead.");
 }
 
+fourOutputs _scaled_dot_product_efficient_attention_batch_rule(
```

Comment:
In order to test this, we should add a new OpInfo that has dropout_p = 0. This will give us automated tests for this operator (nn.functional.scaled_dot_product_attention); you'll be able to run `pytest test/functorch/ -v -k "scaled_dot_product_attention".


There is an existing OpInfo that has dropout_p > 0; our test handler can't handle randomness. You should be able to just copy-paste https://github.com/pytorch/pytorch/blob/b2277075b0fe5cf085d369a313863f64c6fb50c3/torch/testing/_internal/common_methods_invocations.py#L13240 and set dropout_p = 0.